client/src/components/ratingsAndReviews/Ratings.jsx

```diff
@@ -8,19 +8,95 @@ class Ratings extends React.Component {
   }
 
   render() {
+    const { meta } = this.props;
+
+    if (Object.keys(meta).length !== 0) {
+      // find average rating
+      let numOfReviews = 0;
```

Comment:
We can probably move this average rating method up to App to be reused! I'll make a ticket about it, other than that this looks great!